src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Column,
+  GridColumns,
+  Text,
+  Image,
+  Separator,
+  Flex,
+  themeProps,
+} from "@artsy/palette"
+import React from "react"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+export const PriceDatabaseBenefits: React.FC = () => {
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+
+  return (
+    <Flex py={[1, 4]} flexDirection="column">
+      <GridColumns my={4} gridRowGap={[2, 0]}>
+        <Column span={12}>
+          <Text as="h1" variant="xl">
+            The Benefits of the
+            <br />
+            Artsy Price Database
+          </Text>
+        </Column>
+      </GridColumns>
+      <GridColumns mb={4} gridRowGap={[2, 0]}>
+        <Column span={6}>
+          <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
+        </Column>
+        <Column span={6}>
+          <Text as="h1" variant="xl">
+            340,000 Artists
+          </Text>
+          <Text lineHeight={1.5}>
+            From well-known to niche auction houses, find sale results from your
+            favourite artists.
+          </Text>
+        </Column>
+      </GridColumns>
+
+      <Separator />
+
+      <GridColumns my={4} gridRowGap={[2, 0]}>
+        {isMobile && (
+          <Column span={6}>
+            <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
+          </Column>
+        )}
+        <Column span={6}>
+          <Text as="h1" variant="xl">
+            1 Database
+          </Text>
+          <Text lineHeight={1.5}>
```

Comment:
Text always requires a `variant` for the time being (I realize the typing doesn't reflect this). And shouldn't be setting a custom line-height.